js/functions.js

```diff
@@ -0,0 +1,78 @@
+/* Домашнее задание
+Функция №1 для проверки длины строки (2 варианта). */
+
+function checkLength1 (row, maxLen) {
+  console.log(row.length >= maxLen ?  'false' :  'true');
+}
+
+checkLength1('hello my name is kate', 40)
+
+//Вариант 2
+
+function checkLength2 (row, maxLen) {
+  if (row.length <= maxLen) {
+    return true
+  }
+  return false
+}
+
+console.log(checkLength2 ('hi whatsup', 3))
+
+//Функция №2 для проверки палиндромов
+
+function isPal (word) {
+  let newWord = word.split(' ')
```

Comment:
```suggestion
  let newWord = word.toLowerCase();
```

Если я правильно понял, то делаешь массив из строки, чтобы потом по индексу доставать из него элементы `newWord[i]` - это не обязательно делать, так как аналогичный функционал есть и у [строки](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#character_access). Давай просто изначально строку приведем к нижнему регистру?